### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by Unicorn-V** [community/community#116609] [Copilot Conversations]
### Select Topic Area

Question

### Body

I encountered some issues while using the Github Copilot. Specifically, I'm receiving an 'Invalid Copilot token' error message. The error log shows follows:

2024-04-01 20:19:26.645 info auth Invalid copilot token: missing token: 403 
2024-04-01 20:19:26.749 error auth Extension activation failed: "Contact Support. You are currently logged in as Unicorn-V."

I reached out to GitHub Support seeking assistance, only to be informed that my access to Copilot had been revoked due to a purported violation. However, I must assert with all sincerity that my use of Copilot was strictly confined to programming and learning code within VSCode, without any intentional infringement on my part.

![answer](https://github.com/community/community/assets/80693265/a2767262-bcfe-4661-93c5-0ee41ade3f4e)

As a student venturing into the realm of programming, Copilot has been an indispensable tool. I am fully committed to adhering to GitHub's policies and am willing to take any necessary steps to rectify this situation. 

What can I do to reinstate my access to Copilot?

Sincerely, Unicorn-V

**Comment by Akash1134** [comment]
Hi @Unicorn-V ,

As you're already in contact with our Support team, I suggest reaching out to them regarding the reinstatement of your account. Unfortunately, here in an open forum like Community Discussions, we're unable to assist with account-related matters.

Thanks for your understanding.

**Comment by ayim** [comment]
Locking this discussion since it's account related
